src/CampaignsAdmin/CampaignList.tsx

```diff
@@ -38,35 +38,38 @@ export const CampaignList = () => {
       },
     },
     {
-      title: 'Delete',
-      dataIndex: 'delete',
-      key: 'delete',
+      title: 'Actions',
+      dataIndex: 'actions',
+      key: 'actions',
       render: (_: unknown, record: definitions['Campaigns']) => {
         return (
-          <Popconfirm
-            title="Are you sure to delete this campaign 🧐?"
-            onConfirm={async () => {
-              await deleteCampaign(record.id);              
-              triggerGetCampaigns();
-            }}
-            okText="Yes"
-            cancelText="No"
-          >
-            {' '}
-            <Button type="primary" icon={<DeleteOutlined />} size={'small'}>Remove</Button>
-          </Popconfirm>
-        );
-      },
-    },
-    {
-      title: 'Details',
-      dataIndex: 'details',
-      key: 'details',
-      render: (_: unknown, record: definitions['Campaigns']) => {
-        return (
-          <Link target={'_blank'} key={record.id} to={`/CampaignsAdmin/${record.id}`}>
-            <Button type="primary" icon={<EyeOutlined />} size={'small'}></Button>
-          </Link>
+          <Button.Group>
+            <Space>
+              <Link target='_parent'   key={record.id} to={`/CampaignsAdmin/edit/${record.id}`}>
+                <Button type="primary" icon={<EditOutlined />} size={'small'}>
+                  Edit
+                </Button>
+              </Link>
+              <Link target={'_parent'} key={record.id + 1} to={`/CampaignsAdmin/report/${record.id}`}>
```

Comment:
Why is this `record.id + 1`?

Reply:
oh my bad. I was trying something and I forgot change it back